### PR TITLE
Fix aliases output in wbsearchentities

### PR DESCRIPTION
**Pull request by thiemowmde**
[Bug: T110021](https://phabricator.wikimedia.org/T110021)

### CODE REVIEW

**Comment by thiemowmde** [comment]
I honestly have no idea why this complains about a conflict. There is none when I look at this on my machine.

File: src/PropertySuggester/GetSuggestions.php [hunk 1]
```diff
@@ -121,6 +121,7 @@ private function querySearchApi( $resultSize, $search, $language ) {
 				'search' => $search,
 				'action' => 'wbsearchentities',
 				'language' => $language,
+				'uselang' => $language,
```
Comment (thiemowmde) [inline src/PropertySuggester/GetSuggestions.php]: This is an intended change, see [T109584](https://phabricator.wikimedia.org/T109584), but probably should be in it's own patch.

**Comment by addshore** [comment]
General +1 but I would have to test a little before merging probably.....

**Comment by addshore** [comment]
This doesn't seem to limit the number of aliases than can be returned anywhere.

If there is a label that matches the query then we unset the aliases
If there is no label then we set the ID to the label and leave the aliases as they are

Or is that what the magic thing in checkAndSetAlias is doing?
This code could probably do with a good refactor at some point...

**Comment by thiemowmde** [comment]
There is never more than 1 alias because of the check in `checkAndSetAlias`.
